package.json

```diff
@@ -105,6 +105,7 @@
     "moment": "2.10.6",
     "morgan": "1.2.0",
     "ms": "0.7.1",
+    "mustache": "2.3.0",
```

Comment:
Should it be here?

Reply:
You mean it should be declared in `devDependencies` instead?

Reply:
No, it shouldn't be there. As suggested, I removed using a templating engine in favor of using plain text files for that.